clusters/coreos/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+pushd $( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+godep go build -a 
+
+docker build -t vish/heapster-buddy-coreos:canary .
```

Comment:
This is a public repo. Any suggestions for other repos? I can use kubernetes, if you think thats ok.